flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/table/descriptors/ElasticsearchValidator.java

```diff
@@ -146,28 +147,32 @@ private void validateConnectionProperties(DescriptorProperties properties) {
 	 * <pre>
 	 *     connector.hosts = http://host_name:9092;http://host_name:9093
 	 * </pre>
-	 * @param descriptorProperties
-	 * @return
 	 */
-	public static List<ElasticsearchUpsertTableSinkBase.Host> validateAndGetHostsStr(DescriptorProperties descriptorProperties) {
-		final List<ElasticsearchUpsertTableSinkBase.Host> hostList = new ArrayList<>();
+	public static List<Host> validateAndParseHostsString(DescriptorProperties descriptorProperties) {
+		final List<Host> hostList = new ArrayList<>();
 
+		descriptorProperties.validateString(CONNECTOR_HOSTS, false, 1);
 		final String hostsStr = descriptorProperties.getString(CONNECTOR_HOSTS);
-		if (hostsStr.isEmpty()) {
-			throw new ValidationException("Properties '" + CONNECTOR_HOSTS + "' can not be empty.");
-		}
 
 		final String[] hosts = hostsStr.split(";");
 		for (String host : hosts) {
+			final String validationExceptionMessage = "Properties '" + CONNECTOR_HOSTS + "' format should " +
+					"follow the format 'http://host_name:port', but is '" + host + "'.";
 			try {
 				final URL url = new URL(host);
 				final String protocol = url.getProtocol();
-				final String hostNmae = url.getHost();
+				final String hostName = url.getHost();
 				final int hostPort = url.getPort();
-				hostList.add(new ElasticsearchUpsertTableSinkBase.Host(hostNmae, hostPort, protocol));
+
+				if (null == protocol || protocol.isEmpty()
```

Comment:
Can use `org.apache.flink.util.StringUtils#isNullOrWhitespaceOnly` to validate null and empty.

Reply:
ok